packages/scalars/.eslintrc

```diff
@@ -0,0 +1,54 @@
+{
+    "env": {
+        "browser": true,
+        "es2016": true,
+        "node": true,
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/strict-type-checked",
+        "plugin:@typescript-eslint/stylistic-type-checked",
+        "plugin:prettier/recommended",
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module",
+        "tsconfigRootDir": "."
+    },
+    "plugins": ["@typescript-eslint"],
+    "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
Would be cool to create our own rule list so we can just extend it in each project.